src/extensions/default/JavaScriptCodeHints/main.js

```diff
@@ -867,6 +923,8 @@ define(function (require, exports, module) {
 
             return response;
         }
+        
+        CommandManager.register("Rename", "refactor.rename", handleRename);
```

Comment:
You should add the string in `command/Commands.js` instead of using the string explicit.